packages/ra-ui-materialui/src/auth/Login.tsx

```diff
@@ -143,15 +139,13 @@ class Login extends Component {
     }
 }
 
+const Login = withStyles(styles)(LoginView) as ComponentType<Props>;
+
 Login.propTypes = {
```

Comment:
why do you keep the propTypes?

Reply:
As discussed, Props interfaces are for TypeScript compilation, propTypes are for our users who are not using TypeScript.